# 365 Probabilidades — Dia #016
## Qual a probabilidade de recomeçar ser a decisão certa, não o medo?

**Tipo:** Preditivo  
**Data de publicação:** 2026-06-29  
**Ferramenta:** Python  
**Decisão analisada:** Mudar de carreira agora é arriscado, ou é a estabilidade que está me custando caro?  
**Hashtag:** #365Probabilidades #Dia016

---

### 📖 A História

Segunda-feira de manhã é quando a pergunta pesa mais: continuar, ou recomeçar?

A intuição mais comum diz que mudar de carreira é um risco financeiro — que é mais seguro permanecer onde já se está, mesmo insatisfeito, do que arriscar uma transição. Os dados contam uma história mais precisa, e mais interessante: o que realmente determina o resultado financeiro de uma mudança de carreira não é a mudança em si. É se ela foi **escolhida** ou **imposta**.

A Organização para a Cooperação e Desenvolvimento Econômico (OCDE) analisou dados de mercado de trabalho de 17 países ao longo de uma década e encontrou um padrão consistente: quem muda de carreira por escolha própria ganha dinheiro. Quem é forçado a mudar — por demissão, fechamento de empresa, ou outras circunstâncias fora do seu controle — perde.

Não é o recomeço que é arriscado. É a falta de controle sobre ele.

---

### 📚 O Conceito

A literatura de economia do trabalho distingue dois tipos de transição de carreira: **mobilidade voluntária** (a pessoa decide buscar uma nova posição, geralmente porque encontrou algo melhor) e **mobilidade involuntária** (a pessoa é forçada a mudar, por demissão, fechamento da empresa, ou automação do cargo).

Embora ambas resultem na mesma ação observável — trocar de emprego ou de carreira — os resultados financeiros são radicalmente diferentes. A mobilidade voluntária tende a realocar trabalhadores para empresas que pagam melhor e são mais produtivas, gerando ganho real. A mobilidade involuntária acontece sob pressão de tempo e sem poder de negociação, e tende a empurrar a pessoa para posições piores do que a anterior.

---

### 🧮 O Modelo
**Fontes:**
- OECD (2024) — *Promoting Better Career Choices for Longer Working Lives* — dados de mercado de trabalho de 17 países, 2010-2020
- U.S. Bureau of Labor Statistics (2018) — dados de rotatividade de emprego por faixa etária (dado complementar)


In [5]:
import numpy as np
import matplotlib.pyplot as plt
from scipy import stats

plt.rcParams['font.family'] = 'serif'
plt.rcParams['axes.spines.top'] = False
plt.rcParams['axes.spines.right'] = False
plt.rcParams['axes.grid'] = True
plt.rcParams['grid.alpha'] = 0.3
plt.rcParams['figure.facecolor'] = 'white'
plt.rcParams['axes.facecolor'] = 'white'

print('✅ Bibliotecas carregadas')

✅ Bibliotecas carregadas


In [6]:
# --- DADOS DA LITERATURA ---
# OECD (2024) — Promoting Better Career Choices for Longer Working Lives
# Dados de mercado de trabalho de 17 países, periodo 2010-2020

# Crescimento/queda salarial apos mudanca de carreira, por tipo e faixa etaria
ganho_voluntaria_45_54 = 0.074   # +7,4% — mudanca voluntaria, 45-54 anos
ganho_voluntaria_55_64 = 0.035   # +3,5% — mudanca voluntaria, 55-64 anos
perda_forcada_45_54    = -0.09   # -9%   — mudanca forcada, 45-54 anos
perda_forcada_55_64    = -0.13   # -13%  — mudanca forcada, 55-64 anos

# Empregabilidade aos 60 anos, segundo mudanca de carreira entre 45-54 anos
# (dado complementar, ja utilizado no Dia #004 deste projeto, mesma fonte OECD)
p_empregado_60_com_mudanca = 0.62
p_empregado_60_sem_mudanca = 0.54

fator_correcao = 0.80

print('=' * 65)
print('  DADOS DA LITERATURA — RECOMEÇO DE CARREIRA')
print('=' * 65)
print(f'\n  Faixa 45-54 anos:')
print(f'  → Mudança voluntária : {ganho_voluntaria_45_54*100:+.1f}%')
print(f'  → Mudança forçada    : {perda_forcada_45_54*100:+.1f}%')
print(f'\n  Faixa 55-64 anos:')
print(f'  → Mudança voluntária : {ganho_voluntaria_55_64*100:+.1f}%')
print(f'  → Mudança forçada    : {perda_forcada_55_64*100:+.1f}%')
print(f'\n  Empregabilidade aos 60 anos:')
print(f'  → Com mudança entre 45-54 : {p_empregado_60_com_mudanca*100:.0f}%')
print(f'  → Sem mudança              : {p_empregado_60_sem_mudanca*100:.0f}%')
print(f'\n  Fator de correção aplicado: x{fator_correcao}')
print('=' * 65)

  DADOS DA LITERATURA — RECOMEÇO DE CARREIRA

  Faixa 45-54 anos:
  → Mudança voluntária : +7.4%
  → Mudança forçada    : -9.0%

  Faixa 55-64 anos:
  → Mudança voluntária : +3.5%
  → Mudança forçada    : -13.0%

  Empregabilidade aos 60 anos:
  → Com mudança entre 45-54 : 62%
  → Sem mudança              : 54%

  Fator de correção aplicado: x0.8


In [7]:
# --- O MODELO ---
# O modelo central aqui é a DIFERENÇA entre os dois tipos de transição —
# não uma proporção isolada, e sim o contraste entre escolha e imposição.

diferenca_45_54 = ganho_voluntaria_45_54 - perda_forcada_45_54
diferenca_55_64 = ganho_voluntaria_55_64 - perda_forcada_55_64

# Ganho de empregabilidade (dado complementar)
ganho_empregabilidade_pp = (p_empregado_60_com_mudanca - p_empregado_60_sem_mudanca) * 100

# Correção aplicada aos ganhos/perdas centrais
ganho_voluntaria_corrigido_45_54 = ganho_voluntaria_45_54 * fator_correcao
perda_forcada_corrigida_45_54    = perda_forcada_45_54 * fator_correcao

print('=' * 65)
print('  MODELO — RESULTADO')
print('=' * 65)
print(f'  Diferença de impacto salarial — voluntária vs forçada:')
print(f'  → Faixa 45-54 anos: {diferenca_45_54*100:.1f} pontos percentuais de diferença')
print(f'  → Faixa 55-64 anos: {diferenca_55_64*100:.1f} pontos percentuais de diferença')
print()
print(f'  Empregabilidade aos 60 anos:')
print(f'  → Ganho de quem mudou voluntariamente: +{ganho_empregabilidade_pp:.0f}pp')
print()
print(f'  Após fator de correção (×0.80):')
print(f'  → Ganho voluntária corrigido (45-54): {ganho_voluntaria_corrigido_45_54*100:+.1f}%')
print(f'  → Perda forçada corrigida (45-54)   : {perda_forcada_corrigida_45_54*100:+.1f}%')
print('=' * 65)

  MODELO — RESULTADO
  Diferença de impacto salarial — voluntária vs forçada:
  → Faixa 45-54 anos: 16.4 pontos percentuais de diferença
  → Faixa 55-64 anos: 16.5 pontos percentuais de diferença

  Empregabilidade aos 60 anos:
  → Ganho de quem mudou voluntariamente: +8pp

  Após fator de correção (×0.80):
  → Ganho voluntária corrigido (45-54): +5.9%
  → Perda forçada corrigida (45-54)   : -7.2%


In [8]:
# --- VISUALIZACAO ---

# GRAFICO 1 — Voluntária vs Forçada, por faixa etária
fig1, ax1 = plt.subplots(figsize=(12, 8))

categorias = ['Voluntária\n45-54 anos', 'Forçada\n45-54 anos', 'Voluntária\n55-64 anos', 'Forçada\n55-64 anos']
valores = [ganho_voluntaria_45_54 * 100, perda_forcada_45_54 * 100,
           ganho_voluntaria_55_64 * 100, perda_forcada_55_64 * 100]
cores1 = ['#2a8a82', '#c0392b', '#2a8a82', '#c0392b']

bars = ax1.bar(categorias, valores, color=cores1, alpha=0.85, width=0.55)
ax1.axhline(y=0, color='#333333', linewidth=1)
ax1.set_ylabel('Variação salarial após mudança de carreira (%)', fontsize=12)
ax1.set_title('Recomeçar não é o risco — ser forçado a recomeçar é\nOECD (2024) — dados de 17 países, 2010-2020',
              fontsize=13, pad=15)

for bar, v in zip(bars, valores):
    offset = 0.8 if v >= 0 else -1.8
    ax1.text(bar.get_x() + bar.get_width() / 2, v + offset,
             f'{v:+.1f}%', ha='center', fontweight='bold', fontsize=13)

plt.tight_layout()
plt.savefig('dia-016-grafico-01-voluntaria-forcada.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Grafico 1 salvo!')


# GRAFICO 2 — A diferença entre escolher e ser forçado
fig2, ax2 = plt.subplots(figsize=(12, 8))

faixas2 = ['45-54 anos', '55-64 anos']
diferencas = [diferenca_45_54 * 100, diferenca_55_64 * 100]
cores2 = ['#c8a84b', '#c8a84b']

bars2 = ax2.bar(faixas2, diferencas, color=cores2, alpha=0.85, width=0.4)
ax2.set_ylabel('Diferença de impacto salarial (pontos percentuais)', fontsize=12)
ax2.set_title('A diferença entre escolher e ser forçado a recomeçar\nOECD (2024) — mesma ação, resultado oposto',
              fontsize=13, pad=15)
ax2.set_ylim(0, 20)

for bar, d in zip(bars2, diferencas):
    ax2.text(bar.get_x() + bar.get_width() / 2, d + 0.5,
             f'{d:.1f}pp', ha='center', fontweight='bold', fontsize=15)

plt.tight_layout()
plt.savefig('dia-016-grafico-02-diferenca.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Grafico 2 salvo!')


# GRAFICO 3 — Empregabilidade aos 60 anos
fig3, ax3 = plt.subplots(figsize=(12, 8))

categorias3 = ['Sem mudança\nde carreira', 'Com mudança voluntária\nentre 45-54 anos']
valores3 = [p_empregado_60_sem_mudanca * 100, p_empregado_60_com_mudanca * 100]
cores3 = ['#888888', '#2a8a82']

bars3 = ax3.bar(categorias3, valores3, color=cores3, alpha=0.85, width=0.45)
ax3.set_ylim(0, 75)
ax3.set_ylabel('Probabilidade de estar empregado aos 60 anos (%)', fontsize=12)
ax3.set_title('Quem recomeça por escolha tem mais chance de seguir empregado\nOECD — dados longitudinais',
              fontsize=13, pad=15)

for bar, v in zip(bars3, valores3):
    ax3.text(bar.get_x() + bar.get_width() / 2, v + 1.5,
             f'{v:.0f}%', ha='center', fontweight='bold', fontsize=15)

plt.tight_layout()
plt.savefig('dia-016-grafico-03-empregabilidade.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Grafico 3 salvo!')

✅ Grafico 1 salvo!
✅ Grafico 2 salvo!
✅ Grafico 3 salvo!


### 💡 O Insight

Trabalhadores de 45-54 anos que mudaram de carreira **por escolha própria** tiveram crescimento salarial médio de **7,4%**. Quem foi **forçado** a mudar, na mesma faixa etária, teve queda média de **9%**. A diferença entre os dois grupos é de **16,4 pontos percentuais** — a mesma ação observável (trocar de carreira), resultados financeiros opostos.

Na faixa 55-64 anos, o padrão se repete: +3,5% para quem escolheu, -13% para quem foi forçado — uma diferença de **16,5 pontos percentuais**.

E há um ganho adicional, menos falado: quem muda de carreira voluntariamente entre 45 e 54 anos tem **62% de chance de ainda estar empregado aos 60 anos**, contra 54% de quem não fez a mudança — um ganho de 8 pontos percentuais de estabilidade futura.

O medo de recomeçar geralmente mistura duas coisas que são, na verdade, muito diferentes: o risco de mudar, e o risco de ser pego de surpresa por uma mudança que você não escolheu. Os dados mostram que o primeiro, quando vem de uma decisão deliberada, tende a compensar. O segundo é o que realmente custa caro.

Vale notar um dado complementar: pessoas mais velhas trocam de emprego com muito menos frequência que as mais jovens. Segundo o Bureau of Labor Statistics dos EUA, apenas 3,1% dos trabalhadores entre 35 e 54 anos trocaram de emprego em um ano, contra 6,5% dos trabalhadores com menos de 35. O tempo médio no mesmo cargo também cresce com a idade — de 3,2 anos na faixa de 25-34 anos para 10,3 anos a partir dos 65. Isso significa que, justamente na fase em que recomeçar tende a compensar mais financeiramente (a meia-idade), é quando as pessoas recomeçam menos.

*A decisão que você está adiando — ela é movida por medo de recomeçar, ou pelo medo de ser forçada a recomeçar sem ter escolhido?*

---

### ⚠️ Limitações do Modelo
- Os dados são uma média entre 17 países da OCDE com mercados de trabalho e proteções sociais distintas — a realidade brasileira específica pode diferir
- "Mudança voluntária" e "mudança forçada" são categorias que simplificam situações mais complexas — muitas transições têm elementos de ambas
- O ganho ou perda salarial médio esconde grande variação individual entre setores, ocupações e níveis de qualificação
- A correlação entre tipo de mudança e resultado financeiro não exclui completamente fatores de seleção (pessoas com mais recursos podem ter mais condições de escolher quando mudar)
- O fator de correção (×0,80) é uma aproximação padrão do projeto
- O dado complementar do BLS sobre frequência de troca de emprego é americano e de 2018 — usado apenas como contexto, não incorporado ao cálculo central do modelo

*A ciência é honesta sobre o que não sabe. O modelo também.*

---

### 📎 Links
- Substack: [link do post]
- Instagram: [link do post]

---
*365 Probabilidades · Decidindo com dados, um dia de cada vez.*